.circleci/config.yml

```diff
@@ -71,6 +71,7 @@ jobs:
           name: "Bootstrapping"
           command: |
             npm run bootstrap
+            npm run build-packs
```

Comment:
Why run `build-packs` multiple times (why run `bootstrap` multiple times for that matter?)